.cirrus.yml

```diff
@@ -430,7 +381,12 @@ bindings_task:
     env:
         <<: *stdenvars
         TEST_FLAVOR: bindings
-    clone_script: *get_gosrc
+    # N/B: This script depends on ${DISTRO_NV} being defined for the task.
```

Comment:
What value does this comment add?

Reply:
With all the URL-encoding it may be easy for a human to miss the embedded `${DISTRO_NV}` references.  It's also a reminder, that the `env` section (directly or indirectly) must define that value, it's not automatic.

Reply:
For example, in the case of this task, `DISTRO_NV` is set indirectly by the merging of `*stdenvars`

Reply:
I still don't understand. Should you also add "N/B this depends on $ARTCURL and $GOSRC being defined"? Is your intention to say something like "Any task that invokes *get_gosrc must define $DISTRO_NV"? Or "DISTRO_NV must be explicitly defined (or inherited) in any task that runs this script"? If so I don't think the current wording expresses that.